renovate.json

```diff
@@ -20,6 +20,15 @@
       "fileMatch": ["^.github/workflows/continuous-integration.yml$"],
       "matchStrings": ["(?<depName>cypress/included):(?<currentValue>[0-9.]+)\n"],
       "datasourceTemplate": "docker"
+    },
+    {
+      "fileMatch": ["^renovate.json$"],
```

Comment:
:rofl: Renovate kann jetzt seine eigene Config updaten? Skynet confirmed.

Scherz beiseite, bist du sicher, dass `datasourceTemplate` und `versioningTemplate` beide `docker` sein sollten? Ohne die Doku erneut zu lesen hätte ich jetzt angenommen, dass dadurch Renovate ein vollständiges Docker Tag wie z.B. `16.3.0@sha256:e36cf1bb8719551220ba8c3ee1583881e79ad040803570e0849b00b8fe009153` einsetzen wird, liege ich da falsch? Ich hätte jetzt gedacht dass Renovate in den Constraints in seinem Config File einfach SemVer erwartet.

Reply:
Die Doku ist dort bisschen dürftig.
So wie ich das lese war es auch noch nicht gedacht, dass das funktioniert.
Hast du eine möglichkeit gefunden, die renovate config zu testen, (z.b. die dependencies auszugeben, die renovate findet), oder muss man das auf den master bringen, bevor da was geht?

Du hast recht, versioningTemplate sollte es automagically aus der capture group finden.
Das datasourceTemplate ist schwierig, ich habe jetzt mal auf github tags gesetzt. Mal schauen

Reply:
Ich habe keinen wirklich besseren Weg gefunden, Changes zu testen. Bei der Einführung von Renovate hatte ich es auf meinem Fork im master-Branch sowie, und konnte glaube ich so ausprobieren wie es funktioniert. Allenfalls hätten wir noch https://github.com/ecamp/dummy
Die beste Möglichkeit die ich kenne um renovate zu debuggen ist leider, die Logs nach einem Job zu lesen.